airgun/views/host_new.py

```diff
@@ -63,27 +70,108 @@ class Overview(Tab):
 
         @View.nested
         class DetailsCard(Card):
-            ROOT = '(.//article[contains(@class, "pf-c-card")])[1]'
+            ROOT = './/article[.//div[text()="Details"]]'
             details = HostDetailsCard()
 
         @View.nested
         class HostStatusCard(Card):
-            ROOT = '(//article[contains(@class, "pf-c-card")])[2]'
+            ROOT = './/article[.//span[text()="Host Status"]]'
             status = Text('.//h4[contains(@data-ouia-component-id, "OUIA-Generated-Title")]')
 
-            status_success = Text('.//span[@class="status-success"]')
-            status_warning = Text('.//span[@class="status-warning"]')
-            status_error = Text('.//span[@class="status-error"]')
-            status_disabled = Text('.//span[@class="disabled"]')
+            status_success = Text('.//span[span[@class="status-success"]]')
+            status_warning = Text('.//span[span[@class="status-warning"]]')
+            status_error = Text('.//span[span[@class="status-error"]]')
+            status_disabled = Text('.//span[span[@class="disabled"]]')
+
+        @View.nested
+        class InstallableErrataCard(Card):
+            ROOT = './/article[.//div[text()="Installable Errata"]]'
+
+            security_advisory = Text('.//a[contains(@href, "type=security")]')
+            bug_fixes = Text('.//a[contains(@href, "type=bugfix")]')
+            enhancements = Text('.//a[contains(@href, "type=enhancement")]')
+
+        @View.nested
+        class TotalRisksCard(Card):
+            ROOT = './/article[.//div[text()="Total risks"]]'
+
+            low = Text('.//*[@id="legend-labels-0"]/*')
```

Comment:
the correct locator here would be `'.//text[@id="legend-labels-0"]/tspan'`, any suggestions why this is not working? With wildcards, everything is working as expected. Can wildcard stay here as this is matched by id?

Reply:
regarding `'.//text[@id="legend-labels-0"]/tspan'` my guess would be `text` is not a valid tag.

Reply:
tag `text` is probably not the issue here (at least not the only one) as even the `'.//*[@id="legend-labels-0"]/tspan'` is not working.

Reply:
I looked at the locator, it seems to be valid how you have written as it is the react component, there seems to be not located standard way of xpath.